docs/en/api/lynx-testing-environment/index.mdx

```diff
@@ -73,7 +73,7 @@ After configuration, you can directly access the `lynxTestingEnv` object globall
 
 If you want to use `@lynx-js/testing-environment` for unit testing in ReactLynx, you usually don't need to specify this configuration manually.
 
-Please refer to [ReactLynx Testing Library](https://lynxjs.org/react/react-lynx-testing-library.html) to inherit the configuration from `@lynx-js/react/testing-library`.
+Please refer to [ReactLynx Testing Library](https://lynxjs.org/react/reactlynx-testing-library.html) to inherit the configuration from `@lynx-js/react/testing-library`.
```

Comment:
```suggestion
Please refer to [ReactLynx Testing Library](/react/reactlynx-testing-library.html) to inherit the configuration from `@lynx-js/react/testing-library`.
```

Reply:
@colinaaa This file is auto-generated from https://github.com/lynx-family/lynx-stack/blob/a68d10483f52fb6b6b04c3f8484e09621526cd51/packages/testing-library/testing-environment/README.md?plain=1#L66